beluga/include/beluga/sensor/likelihood_field_model.hpp

```diff
@@ -180,19 +175,23 @@ class LikelihoodFieldModel {
 
     const auto neighborhood = [&grid](std::size_t index) { return grid.neighborhood4(index); };
 
+    const auto squared_max_distance = static_cast<float>(params.max_obstacle_distance * params.max_obstacle_distance);
+
     // determine distances to obstacles and calculate likelihood values in-place
     // to minimize memory usage when dealing with large maps
-    auto distance_map = nearest_obstacle_distance_map(grid.obstacle_mask(), squared_distance, neighborhood);
+    auto distance_map = nearest_obstacle_distance_map(
+        grid.obstacle_mask(), squared_distance, neighborhood, static_cast<float>(squared_max_distance));
```

Comment:
@glpuga I think we can spare this `static_cast`, we already cast it to `float` above.

Reply:
True, fixed.